exo-go/src/application/runner.go

```diff
@@ -137,9 +141,11 @@ func (r *Runner) waitForOnlineText(cmdPlus *execplus.CmdPlus, role string, onlin
 	if err != nil {
 		fmt.Printf("'%s' failed to come online after an hour", role)
 	}
-	err = r.Logger.Log(role, fmt.Sprintf("'%s' is running", role), true)
-	if err != nil {
-		fmt.Printf("Error logging '%s' as online: %v\n", role, err)
+	if role != "" {
```

Comment:
I don't know if you remember coming to help me with waiting for text in `docker-compose` log. So we want to wait for `"[service-name] exited with code"` which is output by `docker-compose` itself. So in the log channel, we don't have the prefix `exo-test` yet, therefore the role [here](https://github.com/Originate/exosphere/blob/d3288cbc8662d6c12fef67bf730417703c91ab8f/exo-go/src/application/service_tester.go#L83) needs to be an empty string.